src/components/Modals.js

```diff
@@ -217,3 +217,44 @@ export class SetEncryptKeyModal extends Component {
     );
   }
 }
+
+export class UpdateModelConfirmModal extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {deploy: false};
+  }
+
+  handleDoneAction = (event) => {
+    event?.preventDefault();
+    this.props.doneAction(this.state.deploy);
+  }
+
+  render() {
+    const footer = (
+      <div className="btn-row">
+        <ActionButton type='default'
+          clickAction={this.props.cancelAction} displayLabel={translate('cancel')}/>
+        <ActionButton
+          clickAction={::this.handleDoneAction} displayLabel={translate('ok')}/>
```

Comment:
I always find "Ok" to be a bit unclear in menus. I think changing this to also use "Update" may be more clear.